modules/web/src/app/settings/admin/defaults/template.html

```diff
@@ -278,3 +278,44 @@
     </div>
   </mat-card-content>
 </mat-card>
+
+<mat-card>
+  <mat-card-header>
+    <mat-card-title fxFlex
+                    fxLayout="row"
+                    fxLayoutAlign=" center">
+      <div fxFlex>Provider Defaults</div>
+    </mat-card-title>
+  </mat-card-header>
+  <mat-card-content>
+    <div fxLayout="column"
+         fxLayoutGap="32px"
+         class="admin-settings">
+      <div fxFlex
```

Comment:
```suggestion
  <div fxFlex
           fxLayout="column"
           fxLayoutGap="16px">
```
I don't think so we need these inner div with following layout. Since parent does have already attributes to have a column based layout and inner child's will be always row based.,


But since it used above in template so approving it so that when change is done it is reflected al over the file

Reply:
Just added it for consistency since we are planning on adding more fields in this sub-component.